spec/services/purchase_order_creator_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'rails_helper'
+
+RSpec.describe 'PurchaseOrderCreator' do
+  describe '#call' do
+    let(:cart) { create(:cart) }
+    let(:user) { cart.user }
+    let(:address) { nil }
+    let(:sale) { create(:sale, name: 'Sales free', unit_price_cents: 10) }
+    let!(:cart_item) { create(:cart_item, cart: cart, sale: sale, quantity: 2) }
```

Comment:
considere criar esses items na factory https://github.com/feperessim/refactoring-exercise-for-trainees/blob/master/spec/factories/order_factory.rb

Reply:
Desculpa, factory errada, considere criar esses items nessa factory https://github.com/feperessim/refactoring-exercise-for-trainees/blob/master/spec/factories/cart_factory.rb